packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -234,6 +252,23 @@ abstract class Action<T extends Intent> with Diagnosticable {
       }
     }
   }
+
+  /// Creates an overridable [Action] that allows itself to be overridden by the
```

Comment:
It think it's redundant to say both "overridable [Action]" and "allows itself to be overridden". I'd just remove "overridable".